plugins/value-container-assignment/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+description = "Kotlin Value Container Assignment Compiler Plugin"
```

Comment:
Please also add this change to this commit. It registers testdata for plugin in [TestHelperPlugin](https://github.com/demiurg906/test-data-helper-plugin)
```
Index: .idea/kotlinTestDataPluginTestDataPaths.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/kotlinTestDataPluginTestDataPaths.xml b/.idea/kotlinTestDataPluginTestDataPaths.xml
--- a/.idea/kotlinTestDataPluginTestDataPaths.xml	(revision 0b188f014a898fe799c16517af03b64d6662eb13)
+++ b/.idea/kotlinTestDataPluginTestDataPaths.xml	(date 1662548554411)
@@ -84,6 +84,7 @@
         <option value="$PROJECT_DIR$/plugins/lombok/testData" />
         <option value="$PROJECT_DIR$/plugins/sam-with-receiver/testData" />
         <option value="$PROJECT_DIR$/plugins/kotlinx-serialization/testData" />
+        <option value="$PROJECT_DIR$/plugins/value-container-assignment/testData" />
       </array>
     </option>
   </component>
```

Reply:
Added this line.